client/me/purchases/manage-purchase/index.jsx

```diff
@@ -436,7 +440,7 @@ const ManagePurchase = React.createClass( {
 			<Notice
 				showDismiss={ false }
 				status="is-error"
-				text={ this.translate( 'This purchase has expired and is no longer in use.' ) }>
+				text={ this.props.translate( 'This purchase has expired and is no longer in use.' ) }>
```

Comment:
If we assign `const { translate } = this.props` here and there, we could save some code by avoiding to have to use `this.props.` everywhere.